docs/source/upcoming_release_notes/999-optics-notepad.rst

```diff
@@ -0,0 +1,30 @@
+999 optics-notepad
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- OpticsPitchNotepad
```

Comment:
The last item on the checklist is:
```
Pre-release docs include context, functional descriptions, and contributors as appropriate
```
You should be slightly more helpful here so that someone who reads the release notes has an understanding of what this means.

Reply:
added more info under features

Reply:
I think something like this would be more appropriate, because it's not obvious what your 'feature' is applying to as you are proposing now:

```
OpticsPitchNotepad - a class for storing pitch positions {in a notepad IOC for mirrors x, y, and z}
```